vantage6-server/vantage6/server/model/study.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+from sqlalchemy import Column, String, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+
+from vantage6.server.model.base import Base
+
+
+class Study(Base):
+    """
+    Table that describes which studies are available.
+
+    Studies are subsets of one or more organizations within a collaboration that
+    collaborate on a particular substudy. Studies use the same nodes and data
+    as the collaboration they are part of, and also inherit their collaboration's
+    settings.
+
+    Attributes
+    ----------
+    name : str
+        Name of the study
+    collaboration : :class:`~vantage6.server.model.collaboration.Collaboration`
+        Collaboration that this study is part of
+    organizations :
+            list[:class:`~vantage6.server.model.organization.Organization`]
+        List of organizations that are part of this study
+    tasks : list[:class:`~vantage6.server.model.task.Task`]
+        List of tasks that are part of this collaboration
+    """
+
+    # fields
+    name = Column(String, unique=True)
+    collaboration_id = Column(Integer, ForeignKey("collaboration.id"))
+
+    # relationships
+    collaboration = relationship("Collaboration", back_populates="studies")
+    organizations = relationship(
+        "Organization", secondary="StudyMember", back_populates="studies"
+    )
+    tasks = relationship("Task", back_populates="study")
+
+    def __repr__(self):
+        """
+        Returns a string representation of the study.
+
+        Returns
+        -------
+        str
+            String representation of the study
+        """
+        number_of_organizations = len(self.organizations)
+        number_of_tasks = len(self.tasks)
+        return (
+            "<Study "
+            f"{self.id}: '{self.name}', "
```

Comment:
Also mention the `collaboration_id` (although I dont think we use these representations any more as we dont use the shell? Maybe delete these in the future?)
```suggestion
            "<Study "
            f"{self.id}: '{self.name}', "
            f"part of collaboration: {self.collaboration_id}, "
            f"{number_of_organizations} organization(s), "
            f"{number_of_tasks} task(s)"
            ">"
```

Reply:
I guess we use them sometimes in debugging (at least I do), it's useful to me if `print(study)` gives me some nice info

Reply:
I added this btw